Functions.Templates/Templates/CosmosDBTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "defaultFunctionName": "CosmosTriggerCSharp",
+  "description": "$CosmosDBTriggerCSharp_description",
+  "name": "CosmosDBTrigger",
+  "language": "C#",
+  "filters": [ "cosmosDb" ],
```

Comment:
we don't need to put cosmosDb behind hidekey. Please remove "filters": [ "cosmosDb" ]